src/features/defi/providers/fox-farming/hooks/useFoxFarming.ts

```diff
@@ -74,7 +74,11 @@ export const useFoxFarming = (
           wallet,
         })
 
-        const txid = await buildAndBroadcast({ adapter, buildCustomTxInput })
+        const txid = await buildAndBroadcast({
+          adapter,
+          buildCustomTxInput,
+          receiverAddress: undefined, // no receiver for this contract call
```

Comment:
Ditto q. across all occurrences of this guy

Reply:
Discussed this one out, not doing verifications for contracts for the time being